shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -485,6 +485,14 @@ - (void)setFlutterViewDidRenderCallback:(void (^)(void))callback {
 #pragma mark - Surface creation and teardown updates
 
 - (void)surfaceUpdated:(BOOL)appeared {
+  UIApplicationState appState = [UIApplication sharedApplication].applicationState;
+  // Update surface only when application is active.
+  if (appState == UIApplicationStateActive) {
+    [self _innerSurfaceUpdated:appeared];
+  }
+}
+
+- (void)_innerSurfaceUpdated:(BOOL)appeared {
```

Comment:
Not proper naming for objective-c.  Also, there is no verb in the name so it isn't clear what this method does.